packages/core/src/components/button/_button-group.scss

```diff
@@ -148,7 +148,8 @@ Styleguide components.button-group.css
         @include pt-button-minimal-divider();
         display: inline-block;
         position: absolute;
-        top: 0;
+        top: 10%;
+        bottom: 10%;
```

Comment:
Nope, this becomes too small in the non large use case. Percentage works and is adapting its height no matter the button size